apps/studio/components/layouts/ProjectSettingsLayout/SettingsMenu.utils.tsx

```diff
@@ -57,6 +63,12 @@ export const generateSettingsMenu = (
                 url: `/project/${ref}/settings/integrations`,
                 items: [],
               },
+              {
+                name: `Log Drains`,
+                key: `log-drains`,
+                url: `/project/${ref}/settings/log-drains`,
+                items: [],
+              },
```

Comment:
Log Drains and Data API have now become top-level items. Data API relates to a few services, and we’re looking at integrating Log Drains into Integrations.